### PR TITLE
[PC-10184] redirect to new dns

### PR DESCRIPTION
**Pull request by marionnousvalentin**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-10184

## Checklist

I have:

- [ ] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [ ] Made sure my feature is working on the relevant real / virtual devices.
- [ ] Written **unit tests** for my feature.
- [ ] Written **documentation on Notion** for my feature.
- [ ] Added new reusable components to the AppComponents page and communicate to the team on slack
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** for any added TODO/FIXME \
       (for tech tasks, give the best context about the TODO resolution: what/who/when).

## Screenshots

| \*iOS - [Phone name] | \*Android - [Phone name] | Tablet - [Phone name] | Little - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
|                      |                          |                       |                       |

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn trigger:testing:deploy`

### CODE REVIEW

File: src/api/gen/.swagger-codegen/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-3.0.27
\ No newline at end of file
+3.0.26
```
Comment (antoinewg) [inline src/api/gen/.swagger-codegen/VERSION:1]: pk ce changement ?

Comment (marionnousvalentin) [reply src/api/gen/.swagger-codegen/VERSION:1]: aucune idée, ça a été fait automatiquement, je peux voir avec un dev back

File: scripts/generate_api_client.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 set -e
 
 docker run --rm -v ${PWD}:/local swaggerapi/swagger-codegen-cli-v3 generate \
-        -i https://backend.passculture-testing.beta.gouv.fr/native/v1/openapi.json `# schema location` \
+        -i https://backend.testing.passculture.team/native/v1/openapi.json `# schema location` \
```
Comment (emmajmn) [inline scripts/generate_api_client.sh:5]: c'est ça la nouvelle url au final ? (ils ont changé de ce qui a été dit dans le mail ? )

**Review by emmajmn — APPROVED** [review]

File: src/api/gen/.swagger-codegen/VERSION (hunk 1 above)
Comment (antoinewg) [reply src/api/gen/.swagger-codegen/VERSION:1]: c'est parce que ton docker est pas à jour:
> docker pull swaggerapi/swagger-codegen-cli-v3:latest

File: scripts/generate_api_client.sh (hunk 2 above)
Comment (marionnousvalentin) [reply scripts/generate_api_client.sh:5]: yes, Vincent m'a envoyé les nouvelles DNS

File: src/api/gen/.swagger-codegen/VERSION (hunk 1 above)
Comment (marionnousvalentin) [reply src/api/gen/.swagger-codegen/VERSION:1]: ha !

Comment (marionnousvalentin) [reply src/api/gen/.swagger-codegen/VERSION]: c'est good, j'ai mis à jour

**Review by antoinewg — APPROVED** [review]

**Pull request merged**